hazelcast/core.py

```diff
@@ -412,23 +415,23 @@ class HazelcastJsonValue:
     None values are not allowed.
     """
 
-    def __init__(self, value):
+    def __init__(self, value: typing.Union[str, typing.Any]):
```

Comment:
Can `typing.Union[str, typing.Any]` be simplified to `typing.Any`? or just leave out the annotation altogether?